pymeasure/instruments/agilent/agilentB1500.py

```diff
@@ -164,21 +172,21 @@ def initialize_all_spgus(self):
                 self.add_child(SPGU, channel, collection="spgus", prefix="spgu")
 
     def pause(self, pause_seconds):
-        """Pause Command Execution for given time in seconds (``PA``)
+        """Pause Command Execution for given time in seconds. (``PA``)
 
         :param int pause_seconds: Seconds to pause
         """
         self.write(f"PA {pause_seconds}")
 
     def abort(self):
         """Abort the present operation but channels may still output
-        current/voltage (``AB``)
+        current/voltage. (``AB``)
         """
         self.write("AB")
 
     def force_gnd(self):
-        """Force 0V on all channels immediately. Current Settings can
-        be restored with RZ. (``DZ``)
+        """Force 0V on all channels immediately. Current settings can
```

Comment:
Actually, according to SI, there should be some space between magnitude and units: 0 V.